### ISSUE TITLE
when showPreviewContainer = "false" then not emitted "uploadSuccess"

### ISSUE THREAD

**Issue by MahnazSheikhi** [vugar005/ngx-awesome-uploader#2]
Hi,
I need to show the photos after uploading in some other place, but when I place the showPreviewContainer value false, it does not emitted "uploadSuccess".

**Comment by vugar005** [comment]
Hi 
Thanks for report. Yes, uploadSuccess will only be emitted when showPreviewContainer is enabled. But you can listen to file status in your custom adapter and do whatever you want. For instance,
```
 public uploadFile(fileItem: FilePreviewModel) {
    const form = new FormData();
    form.append('file', fileItem.file);
    const api = 'https://file-picker-demo.free.beeceptor.com';
    const req = new HttpRequest('POST', api, form, {reportProgress: true});
    return this.http.request(req)
    .pipe(
      map( (res: HttpEvent<any>) => {
        if (res.type === HttpEventType.Response) {
           /** ----------  EMIT your event here.  ------------ */ 
          return res.body.id;
        } else {
          return res;
        }
      })
      );
  }
```
I am going to update documentation regarding this in the next version.

**Comment by vugar005** [comment]
Fixed after version 2.0.0 .

**Comment by vugar005** [comment]
Fixed after version 2.0.0 .

**Comment by MahnazSheikhi** [comment]
Thank you for your answer and follow up
